home_lighting.ino

```diff
@@ -109,13 +110,13 @@ void fadeOut(int port, int port_1) {
     analogWrite(port, brightness);
     analogWrite(port_1, brightness);
     brightness -= 1;
-    delay(10);
+    delay(15);
   }
 }
 
-bool isDarkness(int sensorPin)
+bool isBrightly()
 {
-  return digitalRead(sensorPin) == HIGH;
+  return digitalRead(bright_sensor) == LOW;
 }
 
 unsigned long getRemainingTime(unsigned long startTime)
```

Comment:
Remove unsigned keyword from function return type. long type is enough.
Alse remove change `unsigned long remaining` to `long remaining`. 
As a result, `&& remaining < 1000000000 ` is unnecessary.